import axios from 'axios'
import { handleErrors } from '../utils/functions'
const defErrorMessage = 'An error occured processing your current request!'
